crates/primitives/src/genesis.rs

```diff
@@ -292,6 +311,105 @@ pub struct ChainConfig {
     pub clique: Option<CliqueConfig>,
 }
 
+impl ChainConfig {
+    /// Checks if the blockchain is active at or after the Homestead fork block.
+    pub fn is_homestead_active_at_block(&self, block: Option<u64>) -> bool {
```

Comment:
These methods should probably take a `u64` instead of an `Option`

Reply:
Well, I said to myself that this would leave the possibility of a `None` argument but in fact there is no need, if we use these functions it is because we always have a valid block number.